app/assets/javascripts/spam2.js

```diff
@@ -55,6 +55,10 @@ function search_table(filter, url) {
 }
 
 function batch_nav(bulk) {
+	if((bulk == "batch_delete" || bulk == "batch_comment/delete") && !confirm("Are you sure you want to delete the selected nodes?")) {
+	return false;
+}
```

Comment:
HI @VictoriaAjala, can you change this code to this? This is more readable.
```
if(bulk == "batch_delete" || bulk == "batch_comment/delete") {
        let result = confirm("Are you sure you want to delete the selected nodes?");
        if(!result) return false
}
```

Reply:
Okay, I'll do that right away!

Reply:
I have changed the code, @TildaDares please review